### DISCUSSION TITLE
Union mount for dual SD

### DISCUSSION THREAD

**Discussion by bschram** [rg35xx-cfw/rg35xx-cfw.github.io#118] [Ideas]
Thanks so much for all your work on this system!  I've been loving using it for a few weeks now!

I have been doing some research on ability to take advantage of both SD Cards for storing ROMS. While initially I was investigating symlinks or a bind mount, these seem to have quite a few limitations and drawbacks.  An alternative that I've come across is MergerFS as a potential solution for dynamically mounting both SD cards.  

If MergerFS and dependencies can be packaged with the Batocera/Koriki build, users could configure their union mount in /boot/boot-custom.sh. The mount would appear to the system as a single drive with the combined storage size of both SD card partitions, and MergerFS will decide upon write which SD card to write to based on space (or other configured settings).

I'm sure something like this would be low on the priority list currently as you are working on basic functionality to move through beta. Do you think that this may be possible to implement, or am I going down the wrong track? I do not have much experience with buildroot-based systems to know the limitations.

**Comment by acmeplus** [comment]
I'll definitely have a look. I've not used MergerFS in a while, and I'm not completely sure it will work given the stock kernel limitations, but it's definitely something to explore.
